src/vs/editor/contrib/parameterHints/parameterHintsWidget.ts

```diff
@@ -312,7 +312,7 @@ export class ParameterHintsWidget extends Disposable implements IContentWidget {
 		} else if (Array.isArray(param.label)) {
 			return param.label;
 		} else {
-			const idx = signature.label.lastIndexOf(param.label);
+			const idx = signature.label.lastIndexOf(`${param.label},`) ?? signature.label.lastIndexOf(`${param.label})`);
```

Comment:
I don't think the part after  `??` will ever be executed since `lastIndexOf` always returns a number

Reply:
Yes that's correct. I have updated my PR. Also took into account these scenarios for idx. Note that I add one because lookup contains additional character at start.
	                //() -> 0
			//(param) -> 1
			//(param, xparam) -> 1
			//(xparam,param) -> 8
			//(xparam,param,yparam) -> 8
			//(xparam, param,yparam) -> 9
			//(xparam, param) -> 9
			//(xparam,param) -> 9